robots-ev3-module/src/main/java/orwell/tank/hardware/Camera/Camera.java

```diff
@@ -0,0 +1,58 @@
+package orwell.tank.hardware.Camera;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class Camera implements Runnable {
+    private final static Logger logback = LoggerFactory.getLogger(Camera.class);
+    private static final long THREAD_SLEEP_BETWEEN_RUNS_MS = 5;
+
+    private final String startCameraScriptAddress;
+    private final String killCameraScriptAddress;
+    private volatile boolean shouldRun;
+    private Process runningProcess;
+
+    public Camera(final String startCameraScriptAddress, final String killCameraScriptAddress) {
+        this.startCameraScriptAddress = startCameraScriptAddress;
+        this.killCameraScriptAddress = killCameraScriptAddress;
+    }
+
+    public void start() {
+        shouldRun = true;
+        new Thread(this).start();
+    }
+
+    public void stop() {
+        shouldRun = false;
+        if (runningProcess != null) {
+            runningProcess.destroy();
+        }
+        try {
+            Runtime.getRuntime().exec(killCameraScriptAddress);
```

Comment:
To make this kind of calls testable, you may have to provide the runtime in the constructor.
It is not really blocking, so let me know if this is something you will not have time to implement this week (so that I merge anyway).